tools/tool-base/src/test/java/io/spine/tools/protoc/FilePatternsTest.java

```diff
@@ -85,23 +84,23 @@ void regex() {
             void withFileName() {
                 String suffix = "documents.proto";
                 FilePattern filter = FilePatterns.fileSuffix(suffix);
-                assertEquals(suffix, filter.getSuffix());
+                Assertions.assertEquals(suffix, filter.getSuffix());
```

Comment:
Not quite sure why the statically imported methods were replaced with the full references. Was is qualified as a "bad import"? If so, we probably need to address this throughout the project.